microprofile/WFLY-12901_MicroProfile_JWT_Quickstart.adoc

```diff
@@ -0,0 +1,123 @@
+= [WFLY-20901] MicroProfile JWT Quickstart
+:author:            Darran Lofthouse
+:email:             darran.lofthouse@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Under https://issues.jboss.org/browse/WFLY-12609[WFLY-12609] support has been added to the application server 
+for https://microprofile.io/project/eclipse/microprofile-jwt-auth[Eclipse MicroProfile JWT RBAC], this enhancement
+is to provide a Quickstart demonstrating the new functionlity.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-12901[WFLY-12901]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1371[EAP7-1371]
+* https://issues.jboss.org/browse/WFLY-12609[WFLY-12609]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+[ ] Engineering
+
+[ ] QE
+
+=== Affected Projects or Components
+
+=== Other Interested Projects
+
+== Requirements
+
+=== Hard Requirements
+
+A new Quickstart will be developed largely covering the same topics as are covered within the 
+https://quarkus.io/guides/security-jwt[QUARKUS - USING JWT RBAC] Quickstart and generally following a similar flow.  The architectures of Quarkus and WildFly are verry different so the actual order of steps will be adjusted into a natural order for WildFly.
+
+As a pre-requisite it is expected the user will have been able to install and run the latest release of the WildFly
+application server with the `microprofile-jwt-smallrye` subsystem present, no other server side configuration should be required unless nescesary to adapt to a users own environment.
+
+The general steps covered will be: -
+
+1. Generation of a new Maven project using the `archetype:gnerate` goal and a released WildFly archetype.
+
+This step may include some tweaks to the new project to make it suitable for the following steps.
+
+2. Ensuring the project has a base JAX-RS endpoint, deploy the project and call the endpoint using `curl`.
+
+At this stage the project is a simple JAX-RS deployment without security ready for the subsequent steps to 
+enable MicroProfile JWT based security.  The initial endpoint should report the name of the current `Principal` 
+which for now will be `anonymous`. 
+
+3. Generate a suitable public / private key.
+
+As a Quickstart this will be making use of locally generated tokens so local keys will be required.
+
+4. Add a utility to the project to generate Base64 encoded JWT tokens along with the ability to customise some 
```

Comment:
This will be an actual step in the quickstart or it is a feature which will be added with this quickstart?